content/en/account_management/billing/custom_metrics.md

```diff
@@ -5,66 +5,199 @@ aliases:
 
-Integrations limited to 350 custom metrics by default:
+[1]: /developers/metrics/types/?tab=histogram
+[2]: /developers/metrics/types/?tab=histogram#metric-submission-types
+[3]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+{{% /tab %}}
+{{% tab "Distribution" %}}
 
-* [ActiveMQ XML][4]
-* [Go-Expvar][5]
+**A DISTRIBUTION metric generates by default five custom metrics** for each unique combination of metric name, host, and tag values to represent the global statistical distribution of values. These five custom metrics represent server-side aggregations of `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
 
-Integrations with no default limit:
+The number of unique tag value combinations submitted for a DISTRIBUTION metric with this tagging scheme is **four**.
 
-* [Agent Metrics][6]
-* [Directory][7]
-* [Linux Proc Extras][8]
-* [Nagios][9]
-* [PDH Check][10]
-* [Prometheus][11]
-* [SNMP][12]
-* [Windows Services][13]
-* [WMI][14]
+* `host:A`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:400`
+* `host:B`, `endpoint:Y`, `status:200`
 
-Many other integrations can be configured to collect custom metrics, for example:
+The number of custom metrics from a [DISTRIBUTION metric][1] is five times the unique combination of metric name, host, and tag values. This results in `request.Latency` reporting a total of **5*4 = 20 custom metrics**:
 
-* [MySQL][15]
-* [Oracle][16]
-* [Postgres][17]
-* [SQL Server][18]
+##### Customized tagging
+
+You can control over [which tag combination][2] aggregations are performed for any DISTRIBUTION metric. Say that you want to keep only the `endpoint` and `status` tags associated with your distribution. This leaves the following three unique tag combinations:
+
+* `endpoint:X`, `status:200`
+* `endpoint:X`, `status:400`
+* `endpoint:Y`, `status:200`
+
+The number of custom metrics from a [DISTRIBUTION metric][1] is five times the unique combination of metric name, host, and tag values. This results in `request.Latency` reporting a total of **5*3 = 15 custom metrics** with the tagging customisation .
+
+##### Percentile aggregations
+
+**Datadog stores five custom metrics for each potentially queryable tag value combination** to provide you with globally accurate percentiles: `p50`, `p75`, `p90`, `p95`, and `p99`. Suppose you have [added percentile aggregations][3] for `request.Latency` for the set of tags `endpoint` and `status` with the same tag dependency seen earlier. The number of queryable tag value combination is **eight**:
+
+* `endpoint:X`, `status:200`
+* `endpoint:X`, `status:400`
+* `endpoint:Y`, `status:200`
+* `endpoint:X`
+* `endpoint:Y`
+* `status:200`
+* `status:400`
+* `*`
+
+This results in `request.Latency` with percentile aggregation enabled reporting an extra of **5*8 = 40 custom metrics**.
```

Comment:
After enabling percentile aggregations for `request.Latency`, this metric name additionally reports **40** custom metrics.